src/utils/formatDollars.ts

```diff
@@ -1,12 +1,12 @@
 const formatDollars = (amount: any) => {
-  if (amount) {
+  if (amount || amount === 0) {
```

Comment:
```suggestion
  if (amount === 0) {
```

Reply:
I don't think this would handle cases where the values is, e.g., `1200`.

Reply:
OH! I misinterpreted what this line meant.

Reply:
Do you think this would work? I'm just concerned I might misread this again, haha.

```
if (amount >= 0) {
```

Reply:
I think moving the dash handling out of `formatDollars` addressed a lot of the confusion here. `0` is considered falsely in JavaScript, which is why it was originally asking "does this value exist or is it exactly zero".